src/plugins/guided_onboarding/public/plugin.tsx

```diff
@@ -35,7 +33,7 @@ export class GuidedOnboardingPlugin
 {
   constructor() {}
   public setup(
-    core: CoreSetup,
+    _core: CoreSetup,
```

Comment:
Prepending the variable name with an `_` clears it of a warning in the editor (unused variable, in this case)